exercises/concept/take-a-number-deluxe/.docs/introduction.md

```diff
@@ -0,0 +1,147 @@
+# Introduction
+
+## GenServer
+
+~~~~exercism/caution
+This is an advanced topic.
+~~~~
```

Comment:
I added this as a warning because I'm feeling a bit nervous about how big this exercise ended up being. Do you think this is a good idea?

Reply:
Honestly, I don't think it's that advanced. 
I first implemented a `GenServer` on my own in the `bank-account` exercise, and it was very very smooth, the official documentation is very clear for anyone who needs more help.

It may have been complicated to implement `TakeANumberDeluxe.State` but the students don't have to, all they have to do is connect the dots, there is basically no logic to implement.

I think `top-secret` was more complex.